### PR TITLE
actix-router: fix from_hex error

### PR DESCRIPTION
**Pull request by nujz**
## PR Type

Fix

## PR Checklist
Check your PR fulfills the following:

- [x] Tests for the changes have been added / updated.
- [x] Documentation comments have been added / updated.
- [x] A changelog entry has been made for the appropriate packages.
- [x] Format code with the latest stable rustfmt

### CODE REVIEW

**Review by JohnTitor — APPROVED** [review]
Good point, could you also add an entry to the changelog?

**Review by Neopallium — APPROVED** [review]

File: router/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "actix-router"
-version = "0.2.4"
+version = "0.2.5"
```
Comment (JohnTitor) [inline router/Cargo.toml:3]: Do NOT change the version number, we usually do it when releasing.

Comment (robjtede) [reply router/Cargo.toml:3]: on this point, in the changelog, use "unreleased" as the section heading

**Comment by robjtede** [comment]
A newly-passing test for this would be good.

**Comment by nujz** [comment]
I fixed it.

**Comment by robjtede** [comment]
released in 0.2.5

**Pull request merged**